### PR TITLE
DO-40: Added Login API with Dummy token

### PR DESCRIPTION
**Pull request by pepcus-ankit**

### CODE REVIEW

File: src/main/java/com/pepcus/directoryonline/api/controller/AuthenticationController.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+/**
+ * @author ankit.bakliwal
+ * Class for Authentication APIs
+ */
+@RestController
+@RequestMapping(path = "/api/v1/login")
+public class AuthenticationController {
+
```
Comment (shivajain03) [inline src/main/java/com/pepcus/directoryonline/api/controller/AuthenticationController.java:19]: We should not right /api  in every controller. Best practice is to configure that. The way is - server.contextPath = /api in application property.

File: src/main/java/com/pepcus/directoryonline/api/entity/LoginResponse.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.pepcus.directoryonline.api.entity;
+
+public class LoginResponse {
+
```
Comment (shivajain03) [inline src/main/java/com/pepcus/directoryonline/api/entity/LoginResponse.java:4]: Java doc.

File: src/main/java/com/pepcus/directoryonline/api/service/AuthenticationService.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    for(TenantMember member : tenantMembers){
+      Tenant tenantResponse = new Tenant();
+      tenantResponse.setKey(member.getTenant().getKey());
+      tenantResponse.setId(member.getTenant().getId());
+      tenantResponse.setTenantName(member.getTenant().getTenantName());
+      tenants.add(tenantResponse);
+    }
+    response.setTenants(tenants);
+    return response;
+  }
```
Comment (shivajain03) [inline src/main/java/com/pepcus/directoryonline/api/service/AuthenticationService.java:76]: Small chunks should be extracted from this method and sent to loginService, MemberService, tenantService etc..

File: src/main/java/com/pepcus/directoryonline/api/controller/AuthenticationController.java (hunk 1 above)
Comment (pepcus-ankit) [reply src/main/java/com/pepcus/directoryonline/api/controller/AuthenticationController.java:19]: Thank. I will update.

File: src/main/java/com/pepcus/directoryonline/api/service/AuthenticationService.java (hunk 3 above)
Comment (pepcus-ankit) [reply src/main/java/com/pepcus/directoryonline/api/service/AuthenticationService.java:76]: This needs a refactoring, I just added an initial version to unblock frontend. I need to implement the authorization & authentication logic.

File: src/main/java/com/pepcus/directoryonline/api/entity/LoginResponse.java (hunk 2 above)
Comment (pepcus-ankit) [reply src/main/java/com/pepcus/directoryonline/api/entity/LoginResponse.java:4]: My bad. Will make sure to add them in next PR.

**Comment by pepcus-ankit** [comment]
> future

Logging we will introduce. Test not in  MVP product.

**Pull request merged**